crates/common/tedge_config/src/tedge_toml/tedge_config/mapper_config/mod.rs

```diff
@@ -81,40 +82,35 @@ impl DeviceConfig {
 }
 
 /// Bridge configuration fields shared across all cloud types
-#[derive(Debug, Document)]
 pub struct BridgeConfig {
     /// The topic prefix for the bridge MQTT topic
     pub topic_prefix: Keyed<TopicPrefix>,
 
     /// The amount of time after which the bridge should send a ping
     pub keepalive_interval: SecondsOrHumanTime,
+}
 
-    pub include: BridgeIncludeConfig,
+pub struct C8yBridgeConfig {
+    pub include: C8yBridgeIncludeConfig,
```

Comment:
This could have been left as a generic mapper config, no? As it is about the local bridge connection which is valid for any mapper, whether they use it or not?